### PR TITLE
Quadrat: hide query pagination if there's no pagination

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

This change relies on ~https://github.com/WordPress/gutenberg/pull/33635~ https://github.com/WordPress/gutenberg/pull/35092

To test, activate this PR on a completely empty site or one with less than five posts ( so no pagination appears ).

#### Related issue(s):

Solves https://github.com/Automattic/themes/issues/4268

### CODE REVIEW

**Comment by jffng** [comment]
I believe this is ready for a review. 

To test, check out this PR and make sure pagination is appearing correctly. Then, adjust your site so that fewer than five posts exists. No pagination container or dividing line should be rendered.

File: quadrat/block-templates/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!-- wp:template-part {"slug":"header"} /-->
 
-<!-- wp:query {"tagName":"main","layout":{"inherit":true},"queryId":1,"query":{"perPage":5,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":"","inherit":true}} -->
+<!-- wp:query {"tagName":"main","layout":{"inherit":true},"queryId":1,"query":{"perPage":5,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":"","inherit":false}} -->
```
Comment (jffng) [inline quadrat/block-templates/index.html:3]: I had to adjust inherit to false for pagination to work at all... This matches skatepark's index query too: https://github.com/Automattic/themes/blob/trunk/skatepark/block-templates/index.html#L6

**Comment by MaggieCabrera** [comment]
Should we do this on search.html and query-diamond.php too?

**Comment by MaggieCabrera** [comment]
This looks fine to me locally but wpcom still doesn't have the GB change

**Comment by MaggieCabrera** [comment]
Screenshot 2021-10-01 at 10 04 46

I see this on dotcom, don't know if dotcom has the fix or not

**Comment by jffng** [comment]
> Should we do this on search.html and query-diamond.php too?

Oh yeah, I will update there too. 

> I see this on dotcom, don't know if dotcom has the fix or not

Ah I see, shall we hold off on merging until the fix is there?

**Comment by MaggieCabrera** [comment]
Yeah, we talked about holding this one until It lands since the bug is very minor

**Comment by scruffian** [comment]
We should do this for all themes: https://github.com/Automattic/themes/pull/4805

**Pull request closed without merging**